win32ss/user/ntuser/ghost.c

```diff
@@ -161,18 +186,10 @@ BOOL FASTCALL IntMakeHungWindowGhosted(HWND hwndHung)
         return FALSE;   // already ghosting
     }
 
-    // TODO:
-    // 1. Create a thread.
-    // 2. Create a ghost window in the thread.
-    // 3. Do message loop in the thread
-    {
-        static int bWarnedOnce = 0;
-        if (!bWarnedOnce)
-        {
-            bWarnedOnce++;
-            STUB;
-        }
-    }
+    // TODO: Find a way to pass the hwnd of pHungWnd to the ghost thread as we can't pass parameters directly
+
+    if (!gptiGhostThread)
```

Comment:
Do we need to check this under the User lock?

Reply:
Not really